templates/assets/readme-header.html.njk

```diff
@@ -1,7 +1,7 @@
 <link href="font-logos.css" rel="stylesheet">
 
 {% set icons = uniqueIcons %} {# Don't include variants (...-inverse) #}
-{% set rows = 2 %}
+{% set rows = 6 %}
```

Comment:
This is maybe a bit much. We should maybe instead calculate that on a desired number of icons per line? :thinking:

Reply:
Changed.